src/Controllers/HomepageController.php

```diff
@@ -24,7 +24,7 @@ public function __construct(PhpRenderer $renderer, RecipeModel $recipeModel, Ing
 
     public function __invoke(Request $request, Response $response, array $args)
     {
-        if ($_SESSION['loggedIn'] == true) {
+        if (isset($_SESSION['loggedIn']) && $_SESSION['loggedIn'] == true) {
```

Comment:
Seems good and you can remove the == true as it’s truthy I believe

Reply:
Cool I will take a look at this tomorrow